### PR TITLE
Feature: Create totp secret mutation

### PR DESCRIPTION
**Pull request by thiamsantos**
## Description

Add a graphql mutation that will allow the cliente to create a totp secret before the user registration. So the client will be able to setup the 2FA before completing the registration process.

Example mutation:
```graphql
mutation CreateTotpSecret {
  createTotpSecret {
    id
    secret
    recoveryCodes {
      id
      usedAt
    }
  }
}
```

## Changes

- Added a `.tool-versions` file to standardize the versions of elixir and erlang used.
- Added two new database tables: `accounts_recovery_codes` and `accounts_totp_secrets`.
- Added mutation `create_totp_secret`.

### CODE REVIEW

File: lib/invault/accounts/accounts.ex [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      1..@recovery_codes_quantity
+      |> Enum.map(&persist_code(&1, totp_secret))
+
+    %{totp_secret | recovery_codes: recovery_codes}
+  end
+
+  defp persist_code(_, totp_secret) do
+    %RecoveryCode{}
+    |> RecoveryCode.changeset(%{totp_secret_id: totp_secret.id})
+    |> Repo.insert!()
```
Comment (RafaelC457ro) [inline lib/invault/accounts/accounts.ex]: I don't see any transaction here, what happen if some bad thing happen here? Totp Secret inserted before will be reverted?

Comment (thiamsantos) [reply lib/invault/accounts/accounts.ex]: @RafaelC457ro I don't see a huge problem on having a unused secret in the database. But, if you think it is necessary I can add it.

Comment (RafaelC457ro) [reply lib/invault/accounts/accounts.ex]: Just when will perform a insertion.

Comment (thiamsantos) [reply lib/invault/accounts/accounts.ex]: @RafaelC457ro ok.

Comment (thiamsantos) [reply lib/invault/accounts/accounts.ex]: @RafaelC457ro done.

**Pull request merged**